app/assets/javascripts/discourse/app/lib/source-identifier.js

```diff
@@ -30,20 +30,20 @@ export default function identifySource(error) {
 
   let plugin;
 
-  // Source-mapped:
-  plugin = plugin || error.stack.match(/plugins\/([\w-]+)\//)?.[1];
```

Comment:
Adding this in the first place was a mistake - Error#stack in browsers is always based on the 'real' stack, never the sourcemapped stack.